actionpack/lib/abstract_controller/base.rb

```diff
@@ -167,7 +167,7 @@ def performed?
 
     # Returns true if the given controller is capable of rendering
```

Comment:
We have many `Returns true`. Should we replace all `Returns true` to `` Returns `true` ``?

```
% git grep -n 'Returns true'
actionpack/lib/abstract_controller/base.rb:147:    # Returns true if a method for the action is available and
actionpack/lib/abstract_controller/base.rb:168:    # Returns true if the given controller is capable of rendering
actionpack/lib/abstract_controller/base.rb:178:      # Returns true if the name can be considered an action because
actionpack/lib/action_controller/metal/request_forgery_protection.rb:282:      # Returns true or false if a request is verified. Checks:
actionpack/lib/action_controller/metal/strong_parameters.rb:142:    # Returns true if the parameters have no key/value pairs.
actionpack/lib/action_controller/metal/strong_parameters.rb:150:    # Returns true if the given key is present in the parameters.
actionpack/lib/action_controller/metal/strong_parameters.rb:158:    # Returns true if the given value is present for some key in the parameters.
actionpack/lib/action_controller/metal/strong_parameters.rb:166:    # Returns true if the given key is present in the parameters.
actionpack/lib/action_controller/metal/strong_parameters.rb:174:    # Returns true if the given key is present in the parameters.
actionpack/lib/action_controller/metal/strong_parameters.rb:198:    # Returns true if the given value is present for some key in the parameters.
actionpack/lib/action_controller/metal/strong_parameters.rb:240:    # Returns true if another +Parameters+ object contains the same content and
actionpack/lib/action_dispatch/http/request.rb:93:    # Returns true if the request has a header matching the given key parameter.
actionpack/lib/action_dispatch/http/request.rb:245:    # Returns true if the "X-Requested-With" header contains "XMLHttpRequest"
actionpack/lib/action_dispatch/request/session.rb:96:      # Returns true if the session has the given key or false.
activejob/test/support/delayed_job/delayed/backend/test.rb:71:        # Returns true if we have the lock, false otherwise.
activerecord/lib/active_record/associations/association.rb:217:        # Returns true if there is a foreign key present on the owner which
activerecord/lib/active_record/associations/association.rb:250:        # Returns true if inverse association on the given record needs to be set.
activerecord/lib/active_record/associations/association.rb:256:        # Returns true if record contains the foreign_key
activerecord/lib/active_record/associations/association.rb:275:        # Returns true if statement cache should be skipped on the association reader.
activerecord/lib/active_record/associations/collection_association.rb:223:      # Returns true if the collection is empty.
activerecord/lib/active_record/associations/collection_proxy.rb:905:      # Returns true if the collection has more than one record.
activerecord/lib/active_record/attribute_methods.rb:168:      # Returns true if the given attribute exists, otherwise false.
activerecord/lib/active_record/callbacks.rb:307:  # Returns true or false depending on whether the proc is contained in the before_save callback chain on a Topic model.
activerecord/lib/active_record/connection_adapters/abstract/connection_pool.rb:380:      # Returns true if there is an open connection being used for the current thread.
activerecord/lib/active_record/connection_adapters/abstract/connection_pool.rb:416:      # Returns true if a connection has already been opened.
activerecord/lib/active_record/connection_adapters/abstract/connection_pool.rb:903:      # Returns true if there are any active connections among the connection
activerecord/lib/active_record/connection_adapters/abstract/connection_pool.rb:939:      # Returns true if a connection that's accessible to this class has
activerecord/lib/active_record/connection_adapters/abstract_adapter.rb:397:      # Returns true if its required to reload the connection between requests for development mode.
activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb:63:        # Returns true if schema exists.
activerecord/lib/active_record/core.rb:413:    # Returns true if +comparison_object+ is the same exact object, or +comparison_object+
activerecord/lib/active_record/locking/optimistic.rb:142:          # Returns true if the +lock_optimistically+ flag is set to true
activerecord/lib/active_record/persistence.rb:85:    # Returns true if this object hasn't been saved yet -- that is, a record
activerecord/lib/active_record/persistence.rb:92:    # Returns true if this object has been destroyed, otherwise returns false.
activerecord/lib/active_record/persistence.rb:98:    # Returns true if the record is persisted, i.e. it's not a new record and it was
activerecord/lib/active_record/relation.rb:265:    # Returns true if there are no records.
activerecord/lib/active_record/relation.rb:271:    # Returns true if there are no records.
activerecord/lib/active_record/relation.rb:277:    # Returns true if there are any records.
activerecord/lib/active_record/relation.rb:283:    # Returns true if there is exactly one record.
activerecord/lib/active_record/relation.rb:289:    # Returns true if there is more than one record.
activerecord/lib/active_record/relation.rb:591:    # Returns true if relation needs eager loading.
activerecord/lib/active_record/relation.rb:622:    # Returns true if relation is blank.
activerecord/lib/active_record/relation/finder_methods.rb:276:    # Returns true if a record exists in the table that matches the +id+ or
activerecord/lib/active_record/result.rb:71:    # Returns true if there are no records, otherwise false.
activestorage/app/models/active_storage/blob.rb:84:  # Returns true if the content_type of this blob is in the image range, like image/png.
activestorage/app/models/active_storage/blob.rb:89:  # Returns true if the content_type of this blob is in the audio range, like audio/mpeg.
activestorage/app/models/active_storage/blob.rb:94:  # Returns true if the content_type of this blob is in the video range, like video/mp4.
activestorage/app/models/active_storage/blob.rb:99:  # Returns true if the content_type of this blob is in the text range, like text/plain.
activestorage/lib/active_storage/attached/many.rb:28:    # Returns true if any attachments has been made.
activesupport/lib/active_support/cache/memory_store.rb:74:      # Returns true if the cache is currently being pruned.
activesupport/lib/active_support/core_ext/date_and_time/calculations.rb:38:    # Returns true if the date/time is today.
activesupport/lib/active_support/core_ext/date_and_time/calculations.rb:43:    # Returns true if the date/time is in the past.
activesupport/lib/active_support/core_ext/date_and_time/calculations.rb:48:    # Returns true if the date/time is in the future.
activesupport/lib/active_support/core_ext/date_and_time/calculations.rb:53:    # Returns true if the date/time falls on a Saturday or Sunday.
activesupport/lib/active_support/core_ext/date_and_time/calculations.rb:58:    # Returns true if the date/time does not fall on a Saturday or Sunday.
activesupport/lib/active_support/core_ext/load_error.rb:4:  # Returns true if the given path name (except perhaps for the ".rb"
activesupport/lib/active_support/core_ext/numeric/inquiry.rb:5:    # Returns true if the number is positive.
activesupport/lib/active_support/core_ext/numeric/inquiry.rb:14:    # Returns true if the number is negative.
activesupport/lib/active_support/core_ext/object/inclusion.rb:4:  # Returns true if this object is included in the argument. Argument must be
activesupport/lib/active_support/core_ext/string/exclude.rb:4:  # The inverse of <tt>String#include?</tt>. Returns true if the string
activesupport/lib/active_support/logger.rb:12:    # Returns true if the logger destination matches one of the sources
activesupport/lib/active_support/ordered_hash.rb:45:    # Returns true to make sure that this hash is extractable via <tt>Array#extract_options!</tt>
activesupport/lib/active_support/time_with_zone.rb:89:    # Returns true if the current time is within Daylight Savings Time for the
activesupport/lib/active_support/time_with_zone.rb:100:    # Returns true if the current time zone is set to UTC.
activesupport/lib/active_support/time_with_zone.rb:229:    # Returns true if the current object's time is within the specified
activesupport/lib/active_support/time_with_zone.rb:235:    # Returns true if the current object's time is in the past.
activesupport/lib/active_support/time_with_zone.rb:240:    # Returns true if the current object's time falls within
activesupport/lib/active_support/time_with_zone.rb:246:    # Returns true if the current object's time is in the future.
guides/source/action_controller_overview.md:841:| get?, post?, patch?, put?, delete?, head? | Returns true if the HTTP method is GET/POST/PATCH/PUT/DELETE/HEAD.               |
guides/source/api_documentation_guidelines.md:179:# Returns true if the collection is empty.
guides/source/api_documentation_guidelines.md:252:# Returns true if no errors are found, false otherwise.
railties/lib/rails/application.rb:145:    # Returns true if the application is initialized.
railties/lib/rails/command/base.rb:20:        # Returns true when the app is a Rails engine.
```

Reply:
For document, It's be better to be same format as possible as we can.

Reply:
At least `` Returns `true` `` is only 8 lines for now.

```
% git grep -n -i 'Returns `true`'
actionview/app/assets/javascripts/rails-ujs/features/confirm.coffee:13:# Returns `true` if no function stops the chain and user chose yes `false` otherwise.
activerecord/lib/active_record/relation/merger.rb:66:          # `false.blank?` returns `true`, so there needs to be an extra check so that explicit `false` values
guides/source/active_record_querying.md:1836:The above returns `true` if there is at least one client with the `first_name` 'Ryan' and `false`
guides/source/association_basics.md:1421:If the parent object (the one declaring the `has_one` association) is unsaved (that is, `new_record?` returns `true`) then the child objects are not saved. They will automatically when the parent object is saved.
guides/source/association_basics.md:1546:The `collection.empty?` method returns `true` if the collection does not contain any associated objects.
guides/source/association_basics.md:1953:If the parent object (the one declaring the `has_many` association) is unsaved (that is, `new_record?` returns `true`) then the child objects are not saved when they are added. All unsaved members of the association will automatically be saved when the parent is saved.
guides/source/association_basics.md:2076:The `collection.empty?` method returns `true` if the collection does not contain any associated objects.
guides/source/association_basics.md:2319:If the parent object (the one declaring the `has_and_belongs_to_many` association) is unsaved (that is, `new_record?` returns `true`) then the child objects are not saved when they are added. All unsaved members of the association will automatically be saved when the parent is saved.
```

Reply:
I see... On this request, avoiding expand scope of discussion, I hope to deal with only action pack. Is it better to modify more on this request?

Reply:
IMO, "Returns true" is true as Ruby semantics. I'm leave it to other reviewers.